client/usr/lib/systemd/system/prometheus-tunnel-discovery.service

```diff
@@ -4,8 +4,9 @@ After=prometheus-tunnel.service
 BindsTo=prometheus-tunnel.service
 
 [Service]
+EnvironmentFile=/usr/lib/systemd/system/prometheus-tunnel.service.d/tunnel.conf
```

Comment:
This isn't required. See the section on `service.d` in https://www.freedesktop.org/software/systemd/man/systemd.unit.html